File: CS/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.Utils.Win;

namespace DoubledMonthDateEdit
{
    public partial class MainForm : Form
    {
        public MainForm()
        {
            InitializeComponent();
            radioGroupCalendarMode.SelectedIndex = 0;
            radioGroupMultiEditorType.SelectedIndex = 0;
        }


        private void checkEditShowTime_EditValueChanged(object sender, EventArgs e)
        {
            if ((multiDateEditForTest as IPopupControl).PopupWindow != null)
            {
                (multiDateEditForTest as IPopupControl).PopupWindow.Dispose();
            }
            if ((bool)checkEditShowTime.EditValue == true)
            {
                multiDateEditForTest.Properties.VistaEditTime = DevExpress.Utils.DefaultBoolean.True;
            }
            else
            {
                multiDateEditForTest.Properties.VistaEditTime = DevExpress.Utils.DefaultBoolean.False;            
            }
        }

        private void radioGroupMultiEditorType_Properties_EditValueChanged(object sender, EventArgs e)
        {
            if ((multiDateEditForTest as IPopupControl).PopupWindow != null)
            {
                (multiDateEditForTest as IPopupControl).PopupWindow.Dispose();
            }
            multiDateEditForTest.Properties.MultiEditorType = (MultiDateEditor.MultiDateEditOptions)radioGroupMultiEditorType.EditValue;
        }

        private void radioGroupCalendarMode_Properties_EditValueChanged(object sender, EventArgs e)
        {
            if (radioGroupCalendarMode.EditValue.ToString() == "Vista")
            {
                multiDateEditForTest.Properties.VistaDisplayMode = DevExpress.Utils.DefaultBoolean.True;
            }
            else
            {
                multiDateEditForTest.Properties.VistaDisplayMode = DevExpress.Utils.DefaultBoolean.False;
            }
        }
    }
}

File: VB/MainForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Utils.Win

Namespace DoubledMonthDateEdit
	Partial Public Class MainForm
		Inherits Form
		Public Sub New()
			InitializeComponent()
			radioGroupCalendarMode.SelectedIndex = 0
			radioGroupMultiEditorType.SelectedIndex = 0
		End Sub


		Private Sub checkEditShowTime_EditValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles checkEditShowTime.EditValueChanged
			If (TryCast(multiDateEditForTest, IPopupControl)).PopupWindow IsNot Nothing Then
				TryCast(multiDateEditForTest, IPopupControl).PopupWindow.Dispose()
			End If
			If CBool(checkEditShowTime.EditValue) = True Then
				multiDateEditForTest.Properties.VistaEditTime = DevExpress.Utils.DefaultBoolean.True
			Else
				multiDateEditForTest.Properties.VistaEditTime = DevExpress.Utils.DefaultBoolean.False
			End If
		End Sub

		Private Sub radioGroupMultiEditorType_Properties_EditValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles radioGroupMultiEditorType.Properties.EditValueChanged
			If (TryCast(multiDateEditForTest, IPopupControl)).PopupWindow IsNot Nothing Then
				TryCast(multiDateEditForTest, IPopupControl).PopupWindow.Dispose()
			End If
			multiDateEditForTest.Properties.MultiEditorType = CType(radioGroupMultiEditorType.EditValue, MultiDateEditor.MultiDateEditOptions)
		End Sub

		Private Sub radioGroupCalendarMode_Properties_EditValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles radioGroupCalendarMode.Properties.EditValueChanged
			If radioGroupCalendarMode.EditValue.ToString() = "Vista" Then
				multiDateEditForTest.Properties.VistaDisplayMode = DevExpress.Utils.DefaultBoolean.True
			Else
				multiDateEditForTest.Properties.VistaDisplayMode = DevExpress.Utils.DefaultBoolean.False
			End If
		End Sub
	End Class
End Namespace
